environment.yml

```diff
@@ -1,25 +1,419 @@
-# You can use this file to create a conda environment for this pipeline:
-#   conda env create -f environment.yml
-name: nf-core-assemblybacterias-1.0dev
+name: assembly
```

Comment:
@GuilleGorines the env file should only contain direct dependencies as before! Just add minimap2 as dep, not include all your env export!